includes/class-wc-logger.php

```diff
@@ -216,22 +216,28 @@ public function debug( $message, $context = array() ) {
 	 *
 	 * @deprecated 2.7.0
 	 *
+	 * @param string $handle
+	 *
 	 * @return bool
 	 */
-	public function clear() {
+	public function clear( $handle ) {
 		wc_deprecated_function( 'WC_Logger::clear', '2.7', 'WC_Log_Handler_File::clear' );
-		return false;
+		$handler = new WC_Log_Handler_File();
+		return $handler->clear( $handle );
 	}
 
 	/**
 	 * Remove/delete the chosen file.
 	 *
 	 * @deprecated 2.7.0
 	 *
+	 * @param string $handle
+	 *
 	 * @return bool
 	 */
-	public function remove() {
```

Comment:
:man_facepalming: You're right, it was added post `2.6` in 26353d7099d270064bee97ef11acdcd4f6390d13.

👍 Removed.